src/frontend/handlers.go

```diff
@@ -319,6 +338,25 @@ func (fe *frontendServer) placeOrderHandler(w http.ResponseWriter, r *http.Reque
 	}
 }
 
+// postNewRatingHandler posts new rating vote for product.
+// It ignores the error responses on the vote request since it is not critical.
+func (fe *frontendServer) submitRatingHandler(w http.ResponseWriter, r *http.Request) {
+	log := r.Context().Value(ctxKeyLog{}).(logrus.FieldLogger)
+	productID := mux.Vars(r)["id"]
+	if productID == "" {
+		renderHTTPError(log, r, w, errors.New("product id not specified"), http.StatusBadRequest)
+		return
+	}
+	rating, _ := strconv.ParseInt(r.FormValue("rating"), 10, 32)
```

Comment:
what is the meaning of 10 and 32 here? Perhaps, worth adding a comment if GO doesn't support named parameters.

Reply:
I just copy/pasted it from another method in this file:

https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/6eb646640ec7f6f1965329f18cf8ff5ca401e700/src/frontend/handlers.go#L174-L176

Looked for this in the internet and it seems to be a usual [practice](https://golang.org/pkg/strconv/).

Should we deal with this?